lib/cubit/task_cubit.dart

```diff
@@ -16,13 +17,13 @@ class TaskCubit extends Cubit<TaskState> {
     );
   }
 
-  // void readTasks() {
-  //   emit(
-  //     state.copyWith(
-  //       taskNames: [...state.taskNames, response.body],
-  //     ),
-  //   );
-  // }
+  Future<void> readTasks() async {
+    TaskService service = const TaskService();
+    String task1 = await service.readTasks();
```

Comment:
Let's call this variable `tasks` rather than `task1` since you are reading tasks. The fact that it's just one at a time for now is not important. You're setting yourself up for success in the future.